openpype/tools/utils/delegates.py

```diff
@@ -287,9 +287,6 @@ class PrettyTimeDelegate(QtWidgets.QStyledItemDelegate):
     """
 
     def displayText(self, value, locale):
-
-        if value is None:
-            # Ignore None value
-            return
-
-        return pretty_timestamp(value)
+        if value is not None:
+            return pretty_timestamp(value)
+        return None
```

Comment:
With the restructured if statement logic technically this `return None` is redundant too.